game/src/main/kotlin/gg/rsmod/game/sync/segment/PlayerUpdateBlockSegment.kt

```diff
@@ -292,10 +292,9 @@ class PlayerUpdateBlockSegment(val other: Player, private val newPlayer: Boolean
 
             UpdateBlockType.ANIMATION -> {
                 val structure = blocks.updateBlocks[blockType]!!.values
-                buf.put(structure[0].type, structure[0].order, structure[0].transformation, other.blockBuffer.animation)
-                buf.put(structure[1].type, structure[1].order, structure[1].transformation, -1)
-                buf.put(structure[2].type, structure[2].order, structure[2].transformation, -1)
-                buf.put(structure[3].type, structure[3].order, structure[3].transformation, -1)
+                for(i in 0..3) {
+                    buf.put(structure[0].type, structure[0].order, structure[0].transformation, other.blockBuffer.animation)
```

Comment:
Shouldn't this be `structure[i]` or you are meant to send 0 on all of them?

Reply:
It doesn't matter in this particular context, as they're all the same structure (0..3)

This will require refactoring in the future so we can support the animation styles as listed here.

![image](https://user-images.githubusercontent.com/12443214/224571386-1f91e9fa-9f97-47b4-8826-1dceac46183c.png)